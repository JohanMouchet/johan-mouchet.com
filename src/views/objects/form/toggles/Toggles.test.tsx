import React from "react";
import { render } from "@testing-library/react";
import { Checkbox, Radio } from ".";

it("Renders <Checkbox /> without crashing", () => {
  expect(
    render(
      <Checkbox name="name" value="value" checked>
        children
      </Checkbox>
    ).container.firstChild
  ).toMatchSnapshot();
});

it("Renders <Radio /> without crashing", () => {
  expect(
    render(
      <Radio name="name" value="value" checked>
        children
      </Radio>
    ).container.firstChild
  ).toMatchSnapshot();
});
