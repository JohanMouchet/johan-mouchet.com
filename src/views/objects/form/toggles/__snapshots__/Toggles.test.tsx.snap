// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Checkbox /> without crashing 1`] = `
<label
  class="o-form__toggle o-form__toggle--checkbox"
>
  <input
    checked=""
    class="o-form__toggle-input u-is-visually-hidden"
    name="name"
    type="checkbox"
    value="value"
  />
  <span
    class="o-form__toggle-icon"
  />
  <span
    class="o-form__toggle-text"
  >
    children
  </span>
</label>
`;

exports[`Renders <Radio /> without crashing 1`] = `
<label
  class="o-form__toggle o-form__toggle--radio"
>
  <input
    checked=""
    class="o-form__toggle-input u-is-visually-hidden"
    name="name"
    type="radio"
    value="value"
  />
  <span
    class="o-form__toggle-icon"
  />
  <span
    class="o-form__toggle-text"
  >
    children
  </span>
</label>
`;
